WordPress/src/main/java/org/wordpress/android/ui/deeplinks/NotificationsLinkHandler.kt

```diff
@@ -6,24 +6,28 @@ import org.wordpress.android.util.UriWrapper
 import javax.inject.Inject
 
 class NotificationsLinkHandler
-@Inject constructor() {
+@Inject constructor() : DeepLinkHandler {
     /**
      * Builds navigate action from URL like:
      * https://wordpress.com/notifcations
      */
-    fun buildNavigateAction(): NavigateAction {
+    override fun buildNavigateAction(uri: UriWrapper): NavigateAction {
         return OpenNotifications
     }
 
     /**
      * Returns true if the URI should be handled by NotificationsLinkHandler.
      * The handled links are `https://wordpress.com/notifications`
      */
-    fun isNotificationsUrl(uri: UriWrapper): Boolean {
+    override fun shouldHandleUrl(uri: UriWrapper): Boolean {
         return (uri.host == DeepLinkingIntentReceiverViewModel.HOST_WORDPRESS_COM &&
                 uri.pathSegments.firstOrNull() == NOTIFICATIONS_PATH) || uri.host == NOTIFICATIONS_PATH
     }
 
+    override fun stripUrl(uri: UriWrapper): String {
+        return uri.host + "/" + uri.pathSegments.firstOrNull()
```

Comment:
This is missing the handling of the `wordpress://` scheme. Also, when the path segment is null, like on `wordpress://notifications`, we get an actual `null` appended to the end of the URL. Maybe we should handle this like we do in `StatsLinkHandler`?

Reply:
ah you're right 👍 . I missed this usecase